.babelrc

```diff
@@ -25,6 +25,10 @@
 			}
 		]
 	],
+	"overrides": [{
+		"test": "./src/lib/globekit/globekit.esm.js",
+		"compact": true
+	}],
```

Comment:
This didn't have an affect on how the file was bundled. Rather it let's Babel skip the "detection" portion of determining what it should do. We just tell it to compact the file.